sysutils/nut/src/opnsense/www/js/widgets/Nut.js

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (C) 2024 Ben 'DollarSign23'
```

Comment:
```suggestion
 * Copyright (C) 2024 DollarSign23
```

I'll allow a github handle as copyright, but not a mix of name and handle. Would rather see a full name anyway :)